### PR TITLE
feat: Go for firstmile

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes https://github.com/influxdata/ui/issues/4339
Closes https://github.com/influxdata/ui/issues/4340
Closes https://github.com/influxdata/ui/issues/4341
Closes https://github.com/influxdata/ui/issues/4343
Closes https://github.com/influxdata/ui/issues/4350
Closes https://github.com/influxdata/ui/issues/4353
Closes https://github.com/influxdata/ui/issues/4354
Closes https://github.com/influxdata/ui/issues/4355

This PR adds the Go language to the First mile in one go. :)

https://user-images.githubusercontent.com/18511823/162587968-9494352a-0f94-4745-b117-713fd811e036.mov

### CODE REVIEW

File: src/homepageExperience/components/steps/CreateToken.tsx [hunk 1]
```diff
@@ -18,19 +18,15 @@ import {event} from 'src/cloud/utils/reporting'
...
-}
-
-const CreateTokenComponent: FC<ReduxProps & RouteComponentProps> = ({
+const CreateTokenComponent: FC<ReduxProps & RouteComponentProps & OwnProps> = ({
   showOverlay,
   dismissOverlay,
   getAllResources,
+  wizardEventName,
 }) => {
   const org = useSelector(getOrg)
```
Comment (ChitlangeSahas) [inline src/homepageExperience/components/steps/CreateToken.tsx:31]: `CreateToken` is a shared file, so needed the prop to pass in `wizardEventName`

File: src/homepageExperience/containers/NodejsWizard.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export class NodejsWizard extends PureComponent<null, State> {
         return <InstallDependencies />
       }
       case 3: {
-        return <CreateToken />
+        return <CreateToken wizardEventName="nodeJsWizard" />
```
Comment (ChitlangeSahas) [inline src/homepageExperience/containers/NodejsWizard.tsx:79]: Just fixing the event for the python and node wizard

File: src/homepageExperience/components/steps/CreateToken.tsx (hunk 1 above)
Comment (hoorayimhelping) [reply src/homepageExperience/components/steps/CreateToken.tsx:31]: good catch! 🎣

File: src/homepageExperience/containers/HomepageContainer.tsx [hunk 3]
```diff
@@ -43,7 +43,7 @@ export const HomepageContainer: FC = () => {
   const cliPageLink = `/orgs/${org.id}/load-data/file-upload/csv`
   const telegrafPageLink = `/orgs/${org.id}/load-data/telegrafs`
   const javaScriptNodeLink = `/orgs/${org.id}/new-user-wizard/nodejs`
-  const golangLink = `/orgs/${org.id}/load-data/client-libraries/go`
+  const golangLink = `/orgs/${org.id}/load-data/new-user-wizard/nodejs`
```
Comment (hoorayimhelping) [inline src/homepageExperience/containers/HomepageContainer.tsx]:
```suggestion
  const golangLink = `/orgs/${org.id}/load-data/new-user-wizard/golang`
```

File: src/homepageExperience/containers/NodejsWizard.tsx (hunk 2 above)
Comment (hoorayimhelping) [reply src/homepageExperience/containers/NodejsWizard.tsx:79]:
```suggestion
        return <CreateToken wizardEventName="nodejsWizard" />
```

small, but to keep things consistent.

**Review by hoorayimhelping — APPROVED** [review]
go(lang), go(lang), go(lang)!!!!

one small formatting nitpick

File: src/homepageExperience/containers/NodejsWizard.tsx (hunk 2 above)
Comment (ChitlangeSahas) [reply src/homepageExperience/containers/NodejsWizard.tsx]: ✅

**Pull request merged**